shell/src/main/java/alluxio/extension/command/LsCommand.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.extension.command;
+
+import alluxio.Configuration;
+import alluxio.PropertyKey;
+import alluxio.extension.ExtensionUtils;
+import alluxio.util.ShellUtils;
+
+import org.apache.commons.cli.CommandLine;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.util.List;
+
+import javax.annotation.concurrent.ThreadSafe;
+
+/**
+ * Install a new extension.
+ */
+@ThreadSafe
+public final class LsCommand extends AbstractExtensionCommand {
+  private static final Logger LOG = LoggerFactory.getLogger(LsCommand.class);
+
+  /**
+   * Constructs a new instance of {@link LsCommand}.
+   */
+  public LsCommand() {}
+
+  @Override
+  public String getCommandName() {
+    return "ls";
+  }
+
+  protected int getNumOfArgs() {
+    return 0;
+  }
+
+  @Override
+  public String getUsage() {
+    return "ls";
+  }
+
+  @Override
+  public String getDescription() {
+    return "Lists all installed extensions.";
+  }
+
+  @Override
+  public int run(CommandLine cl) {
+    try {
+      String extensionDir = Configuration.get(PropertyKey.EXTENSION_DIR);
```

Comment:
would it make more sense to list the extensions installed locally, vs trying to look up a master and say what's installed there? admins should have access to be able to run commands from the master